stellar-multitenant/pkg/internal/provisioning/manager_test.go

```diff
@@ -153,6 +167,7 @@ func Test_Manager_ProvisionNewTenant(t *testing.T) {
 	})
 
 	messengerClientMock.AssertExpectations(t)
+	distAccSigClientMock.AssertExpectations(t)
```

Comment:
FYI: mocks created with Mockery already call AssertExpectations automatically, like in:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/c1d1af01125f2a26efb16c4abef5fd785bdb0141/internal/transactionsubmission/engine/signing/mocks/signature_client.go#L157-L165

Reply:
cool!